src/server/routes.ts

```diff
@@ -1,9 +1,27 @@
 import express from "express";
+import { fetchGitHubSearchCodeWithUser } from "./gitHubSearch";
 
 const router = express.Router();
 
-router.get("/search", (_req, res) => {
-  res.status(404).send("Hello World");
+router.get("/search", async (_req, res, next) => {
```

Comment:
```suggestion
router.get("/search", async (req, res, next) => {
```

The underscore is only required for unused parameters.